modules/oauth/src/main/resources/META-INF/mule-oauth2.xsd

```diff
@@ -65,6 +66,13 @@
                             </xsd:documentation>
                         </xsd:annotation>
                     </xsd:element>
+                    <xsd:element ref="spring:property" minOccurs="0" maxOccurs="unbounded">
```

Comment:
This Is fine for 3.x, but for sustaining branches spring properties must be used, so the schema is not modified.

Reply:
Sure. I have created other PR for sustaining branches. Please see: https://github.com/mulesoft/mule/pull/6247

Reply:
there shouldn be any schema changes in this pr then.

Reply:
It is not possible to put spring properties here, without adding this element in the schema.
At least in grant type element,, the spring properties are not inherited.